index.html

```diff
@@ -10,12 +10,18 @@
 
   <body>
     <header class="d-flex">
-      <a href="#" class="name">Victor</a>
-      <nav>
-        <img src="pictures/Enabled.png" alt="menu" />
-      </nav>
+      <div class="left-side-head">
+        <a href="#" class="name">Victor Toraño</a>
+      </div>
+      <div class="righ-side-head">
+        <a href="#" class="name-desk">Porfolio</a>
+        <a href="#" class="name-desk">About</a>
+        <a href="#" class="name-desk">Contact</a>
+        <img src="pictures/Icon-Mail.png" class="name-desk" alt="" />
+        <img src="pictures/Enabled.png" class="only-mob" alt="menu" />
```

Comment:
### OPTIONAL
- [ ] Hamburger icon should appear only in mobile version. You can fix it.
- [ ]  when ı change screen width ı ı can scroll it horizontally. You can fix it too. Page should fit when you change screen width.
![image](https://user-images.githubusercontent.com/38635158/126018974-f57db43b-d34c-41fe-8017-d622de8de7f4.png)